Dockerfile

```diff
@@ -43,6 +43,9 @@ ARG PUBLIC_BASE_URL
 ARG MULTIPLAYER_API_URL
 ARG MULTIPLAYER_API_ACCESS_TOKEN
 
+ARG MEILISEARCH_HOST
+ARG MEILISEARCH_API_KEY
+
```

Comment:
Can this also appear in the `.dev` version of this file and point to somewhere remote? If so I think we should add it! Can it also be added to the `.env.template`?

Reply:
forgot about both of these, good catch! will add